payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/resources/LpmRepository.kt

```diff
@@ -615,6 +610,9 @@ class LpmRepository constructor(
         val resources: Resources?,
         val isFinancialConnectionsAvailable: IsFinancialConnectionsAvailable =
             DefaultIsFinancialConnectionsAvailable(),
+        // Whether to enable ACHv2 in the deferred flow.
+        // To be deleted when https://jira.corp.stripe.com/browse/BANKCON-6731 is completed.
+        val enableACHV2InDeferredFlow: Boolean = false,
```

Comment:
Nit: Should we use `BuildConfig.DEBUG` so we can test things in our dev builds?

Reply:
Yea we can do that